src/main/java/org/ibp/api/brapi/v1/study/StudyResourceBrapi.java

```diff
@@ -105,39 +107,73 @@
+		@ApiParam(value = "Filter by season.") @RequestParam(value = "seasonDbId", required = false) final String seasonDbId,
 		@ApiParam(
-			value = "Studies are contained within a trial.  Provide the db id of the trial to list summary of studies within the trial. "
-				+ "Use <code>GET /{crop}/brapi/v1/trials</code> service to retrieve trial summaries first to obtain trialDbIds to supply here. ",
-			required = true) @RequestParam(value = "trialDbId", required = false) final String trialDbId,
+			value =
+				"Filter by trial.") @RequestParam(value = "trialDbId", required = false) final String trialDbId,
+		@ApiParam(value = "Filter by study DbId") @RequestParam(value = "studyDbId", required = false) final String studyDbId,
+		@ApiParam(value = "Filter active status true/false.") @RequestParam(value = "active", required = false) final Boolean active,
+		@ApiParam(value = "Name of the field to sort by.") @RequestParam(value = "sortBy", required = false) final String sortBy,
+		@ApiParam(value = "Sort order direction. Ascending/Descending.") @RequestParam(value = "sortOrder", required = false)
+		final String sortOrder,
 		@ApiParam(value = BrapiPagedResult.CURRENT_PAGE_DESCRIPTION, required = false) @RequestParam(value = "page",
 			required = false) final Integer currentPage,
 		@ApiParam(value = BrapiPagedResult.PAGE_SIZE_DESCRIPTION, required = false) @RequestParam(value = "pageSize",
 			required = false) final Integer pageSize) {
 
-		/***
-		 * Study in BrAPI land = Environment/Instance in BMS/Middleware land. We need to build services in Middleware to list all
-		 * Environment/Instance metadata for a given trialDbId (required parameter). BrAPI does not yet have the trialDbId parameters but we
-		 * need it here so we are going to add it and implement anyway. In future we may support taking array of trial Ids to list studies
-		 * for. There is no point listing all instance/environment summaries across all trials.
-		 *
-		 * studyDbId in BrAPI will map to nd_geolocation_id in Middleware.
-		 *
-		 * For now, just returning an empty place holder message with status. For posterity, keep in mind that BRAPI paging starts at index
-		 * 0.
-		 */
-
-		final Result<org.ibp.api.brapi.v1.study.StudySummaryDto> results = new Result<>();
-		final Pagination pagination = new Pagination();
-		final Metadata metadata = new Metadata().withPagination(pagination)
-			.withStatus(Collections.singletonList(Maps.newHashMap(ImmutableMap.of("message", "This call is not yet implemented."))));
-		final StudySummariesDto studiesList = new StudySummariesDto().setMetadata(metadata).setResult(results);
-
-		return new ResponseEntity<>(studiesList, HttpStatus.OK);
+		final boolean isSortOrderValid = "ASC".equals(sortOrder) || "DESC".equals(sortOrder) || StringUtils.isEmpty(sortOrder);
+		Preconditions.checkArgument(isSortOrderValid, "sortOrder should be either ASC or DESC");
+
+		final StudySearchFilter studySearchFilter = new StudySearchFilter();
```

Comment:
Better create a `StudySearchFilter` constructor with the expected fields from parameter